### PR TITLE
Refactor/storybook cleanup

### PR DESCRIPTION
**Pull request by Jamerrone**
- [Trello](https://trello.com/c/frHXpXPA/77-3-clean-up-guide)
- [Preview on page](https://deploy-preview-98--leanwebkit.netlify.com/guide/)

### CODE REVIEW

File: src/client/components/contact-form/readme.md [hunk 1]
```diff
@@ -2,21 +2,24 @@
 
 A simple contact form to send a message to the recipient and to the person's email who filled in the form.
 
+## Supported input fields
```
Comment (jbmoelker) [inline src/client/components/contact-form/readme.md]: just remove it

File: src/client/components/button-group/button-group.stories.js [hunk 2]
```diff
@@ -1,36 +1,24 @@
...
-      })
-  }))
-  .add('Multiple buttons', () => ({
+  .add('Default', () => ({
     components: {
       ButtonGroup
     },
     data: () => ({
-      items: [
+      singleBtnItem: [
```
Comment (jbmoelker) [inline src/client/components/button-group/button-group.stories.js]: avoid abbrevations (`singleItem` or `singleButton`, etc)

File: src/client/components/social-share/readme.md [hunk 3]
```diff
@@ -1,22 +1,9 @@
...
-
-### Props:
-
-| Name        |   Type   |  Required  |
-|:-----------:|:--------:|:----------:|
-| url         |  string  |   true     |
-| title       |  string  |   true     |
-| description |  string  |   false    |
\ No newline at end of file
+- Works without JavaScript.
```
Comment (jbmoelker) [inline src/client/components/social-share/readme.md]: move to bottom of the list, doesn't convince a product owner

File: src/client/components/table-of-contents/readme.md [hunk 4]
```diff
@@ -1,12 +1,8 @@
...
-### Props:
-
-| Name      |   Type   |  Required  |
-|:---------:|:--------:|:----------:|
-| items     |  array   |    true    |
\ No newline at end of file
+- Create a link for each section on a page.
+- Clicking on a link will scroll the page to the given section.
+- Has a build in scroll spy that highlights the current visible/active section.
+- Is always visible and will scroll with the page.
```
Comment (jbmoelker) [inline src/client/components/table-of-contents/readme.md]: people know the word "sticky" I think

**Review by jbmoelker — DISMISSED** [review]

**Comment by Jamerrone** [comment]
Fixed multiple things based on feedback and merged some conflicts. @jbmoelker

**Review by jbmoelker — APPROVED** [review]

**Pull request merged**